### PR TITLE
Updated db schema path

### LINKED ISSUE #4239: There is path issue on document
# Bug report
There is file path issue.

### Steps to reproduce

1. Go to this link https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/db-schema.html
2. <Magento_root_directory/setup/src/Magento/Setup/Model/Declaration/Schema/ - I didn't get on file system

## Expected result
N/A

## Possible solutions
- Update as per file system

### PR DESCRIPTION
**Pull request by sanganinamrata**
## Purpose of this PR

This PR is updated db_schema's top-level node path which fixes #4239 

## Affected URLs

- https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/db-schema.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ A column can have the following attributes:
 </tr>
 </table>
 
-For more information about each type, refer to the annotations in the corresponding XSD file. The XSD files are located in the `<Magento_root_directory/setup/src/Magento/Setup/Model/Declaration/Schema/etc` directory.
+For more information about each type, refer to the annotations in the corresponding XSD file. The XSD files are located in the `<Magento_root_directory/vendor/magento/framework/Setup/Declaration/Schema/etc` directory.
```
Comment (rogyar) [inline guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md]: Hi @sanganinamrata. Please, add also:

"or `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc`". 

There are two possible places where the mentioned XSD files can be located. It depends on how Magento is installed.

Comment (sanganinamrata) [reply guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md]: @rogyar Sure, I'll do it and let you know.

Comment (sanganinamrata) [reply guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md]: @rogyar I have updated.

File: guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md [hunk 2]
```diff
@@ -148,8 +148,7 @@ A column can have the following attributes:
 </tr>
 </table>
 
-For more information about each type, refer to the annotations in the corresponding XSD file. The XSD files are located in the `<Magento_root_directory/setup/src/Magento/Setup/Model/Declaration/Schema/etc` directory.
-
+For more information about each type, refer to the annotations in the corresponding XSD file. The XSD files are located in the `<Magento_root_directory/setup/src/Magento/Setup/Model/Declaration/Schema/etc` or `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc` directory. There are two possible places where the mentioned XSD files can be located. It depends on how Magento is installed.
```
Comment (jeff-matthews) [inline guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md]: Please correlate each path with a specific installation method.

**Comment by lorikrell** [comment]
@sanganinamrata Hello! If you could check the last request from Jeff, we can complete reviews. Thanks!

File: guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md (hunk 2 above)
Comment (sanganinamrata) [reply guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md]: @jeff-matthews I have updated, Can you please proceed this PR?

File: guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md [hunk 3]
```diff
@@ -148,8 +148,7 @@ A column can have the following attributes:
 </tr>
 </table>
 
-For more information about each type, refer to the annotations in the corresponding XSD file. The XSD files are located in the `<Magento_root_directory/setup/src/Magento/Setup/Model/Declaration/Schema/etc` directory.
-
+For more information about each type, refer to the annotations in the corresponding XSD file. The XSD files are located in the `<Magento_root_directory/vendor/magento/framework/Setup/Declaration/Schema/etc` (If you have downloaded Magento from here https://magento.com/tech-resources/download) or `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc` directory (If you have installed via Github https://github.com/magento/magento2 or using composer). There are two possible places where the mentioned XSD files can be located. It depends on how Magento is installed.
```
Comment (jeff-matthews) [inline guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md]:
```suggestion
For more information about each type, refer to the annotations in the corresponding XSD file. The location of the XSD file depends on how you installed Magento.

- [Archive download]({{page.baseurl}}/install-gde/prereq/zip_install.html): `<Magento_root_directory/vendor/magento/framework/Setup/Declaration/Schema/etc` 
- [Composer]({{page.baseurl}}/install-gde/composer.html) or [GitHub]({{page.baseurtl}}/install-gde/prereq/dev_install.html) installation: `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc`
```

**Comment by sanganinamrata** [comment]
@jeff-matthews I have committed your suggestion.

**Pull request merged**